desktop/WalletManager.ts

```diff
@@ -125,6 +148,84 @@ class WalletManager {
   activateAccounts = (accounts: Account[]) => {
     this.txManager.setAccounts(accounts);
   };
+
+  requestActivationsByCoinbase = async (
+    coinbase: Uint8Array
+  ): Promise<Activation[]> => {
+    const BATCH_SIZE = 50; // the same inside `requestMeshActivations`
```

Comment:
is it possible to reuse this constant, if we 're using same in requestMeshActivations